backend/apps/matching/templates/registration/logout.html

```diff
@@ -21,7 +21,7 @@
 	}
 
 </style>
-<script type="text/javascript">
+<script>
```

Comment:
Out of curiosity, why is this removed/what does it normally do?

Reply:
As per https://javascript.info/hello-world (probably one if not the best JS-resource I know, except for the MDN reference) both type and language attributes of the script tag are no longer required. So in essence it doesn't do anything. 

See also https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script :  
>The HTML5 specification urges authors to omit the attribute rather than provide a redundant MIME type.

There are still uses for the type attribute (using modules in Javascript, which is supported by most modern but not all of the used browsers, one of the reasons to use webpack) and there are several other interesting attributes to the tag. In our case none of them are needed.